Add Helm Chart testing scripts (#32)
kyma-project/helm-broker: test/charts/README.md

> # Charts Tests / Smoke?

Author: ```suggestion
# Charts Tests
```

A chart directory contains tests that validate that our charts work as expected when they are installed. These tests also help the chart consumer understand what your chart is supposed to do.

> A test in a helm chart lives under the `templates/tests` directory and is a pod definition that specifies a container with a given command to run. The container should exit successfully (exit 0) for a test to be considered a success. The pod definition must contain the helm `helm.sh/hook: test-success` hook annotations.

Reviewer: ```suggestion
Find the definition of a Pod that runs tests in the `templates/tests` directory. The Pod definition contains the Helm `helm.sh/hook: test-success` hook annotation, which means that the Pod is executed only when you run the `helm test {relase_name}` command.
```

> You can run the pre-defined tests in Helm on a release using the command `helm test <RELEASE_NAME>`. For a chart consumer, this is a great way to sanity check that their release of a chart (or application) works as expected.

Reviewer: ```suggestion
```
